accesshandler/pkg/providerregistry/registry.go

```diff
@@ -43,6 +44,12 @@ func (pr ProviderRegistry) All() map[string]RegisteredProvider {
 func Registry() ProviderRegistry {
 	return ProviderRegistry{
 		Providers: map[string]map[string]RegisteredProvider{
+			"commonfate/flask": {
+				"v1": {Provider: &flask.Provider{},
```

Comment:
Let's mark this as an alpha version too

```suggestion
				"v1-alpha1": {Provider: &flask.Provider{},
```